lib/frontend/github-client-middleware.js

```diff
@@ -46,4 +47,15 @@ function isAdmin(githubClient) {
   };
 }
 
+function getSignedJsonWebToken({ appId, privateKey }) {
+  const now = Math.floor(Date.now() / 1000);
+  const payload = {
+    iat: now,
+    exp: now + 60 * 10 - 30,
+    iss: appId,
+  };
+  const token = sign(payload, privateKey, { algorithm: 'RS256' });
+  return token;
+}
+
```

Comment:
instead of using the `jsonwebtoken` module you can use `githubClient.auth({ type: "app" })` which will give you
a [JSON Web Token (JWT) authentication](https://github.com/octokit/auth-app.js/#json-web-token-jwt-authentication-1) object.